pkgs/development/tools/alire/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, lib, fetchFromGitHub, gnat, gprbuild }:
+
+stdenv.mkDerivation {
+  pname = "alire";
+  version = "1.3-dev";
```

```suggestion
  version = "1.3-dev-2022-06-24";
```
Since version 1.3-dev is not released version, the commit date should be included.
> If a package is not a release but a commit from a repository, then the version attribute must be the date of that (fetched) commit. The date must be in "unstable-YYYY-MM-DD" format.
https://nixos.org/manual/nixpkgs/unstable/#sec-package-naming